core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -23,9 +23,7 @@
       </li>
     </ul>
 
-    <button type="button" class="btn btn-default full-editor-btn" ng-click="setEditorModeToFull()">
-      Use Advanced Editor
-    </button>
+    <span class="full-editor-switch" ng-click="setEditorModeToFull()">Use Advanced Editor</span>
```

Comment:
For accessibility purposes, I think it's important that this remains as a button. A span does not semantically convey the element's meaning.

I think it is also OK for the element to look like a button, as it used to before. It is a different type of "thing" from all the other options on the sidebar. I also suggest moving the button up a little bit -- currently it's a bit too close to the bottom border.